packages/fxa-content-server/app/styles/modules/_settings.scss

```diff
@@ -781,3 +782,26 @@ section.modal-panel {
     padding-left: 25px;
   }
 }
+
+.breadcrumbs {
+  width: 100%;
+  margin-top: 0px;
+  margin-left: 100px;
+
+  li {
+    list-style: none;
+    float: left;
+    margin: 0px;
+    padding: 0px;
+
+    &:before {
+      content: "\003E";
+      padding: 0px 10px;
+    }
+
+    &:first-child:before {
+      content: "";
+      padding: 0px;
+    }
+  }
+}
```

Comment:
The Github syntax highlighting is suggesting that you don't need `px` after the `0`.  😄

Reply:
Or not.  I just don't understand colors apparently.